site/docs/tools/deltalake/spark.md

```diff
@@ -3,9 +3,7 @@
 !!! note
     Detailed steps on how to set up Pyspark + Delta Lake + Nessie with Python is available on [Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=notebooks/nessie-delta-demo-nba.ipynb).
 
-To access Nessie from a spark cluster make sure the `spark.jars` spark option is set to include
-the [Spark 2](https://repo.maven.apache.org/maven2/org/projectnessie/nessie-deltalake-spark2/{{ versions.java}}/nessie-deltalake-spark2-{{ versions.java}}.jar)
-or [Spark 3](https://repo.maven.apache.org/maven2/org/projectnessie/nessie-deltalake-spark3/{{ versions.java}}/nessie-deltalake-spark3-{{ versions.java}}.jar)
+To access Nessie from a spark cluster make sure the `spark.jars` spark option is set to include the [Spark 3](https://repo.maven.apache.org/maven2/org/projectnessie/nessie-deltalake/{{ versions.java}}/nessie-deltalake-{{ versions.java}}.jar)
```

Comment:
just to clarify: Is Spark 2 no longer supported?

Reply:
I think so, is it @snazy ? If yes, is it also applicable for Iceberg as I see in the docs we have a section about Spark 2, I wonder if we clean that up as well  🤔

Reply:
Yea, only Deltalake 1.0 w/ Spark 3 is supported. We had clients for DL 0.6 w/ Spark2 + DL 0.8 w/ Spark 3 earlier.
ALl mentions of Deltalake w/ Spark 2 can go away.

Reply:
Note: maybe use `[Nessie Deltalake Client for Spark 3]` instead `[Spark 3]` for the link?

Reply:
+1 to link renaming

Reply:
Fix that as well removed other Spark 2 instances in the docs in regards to Delta Lake